horovod/runner/gloo_run.py

```diff
@@ -180,6 +186,9 @@ def _exec_command(command, slot_info, events):
                 stdout_file.close()
             if stderr_file:
                 stderr_file.close()
+        if exit_code != 0 and crash_event and not any(ev.is_set() for ev in all_events):
+            print('Terminating remaining workers after failure of Process {idx}.'.format(idx=index))
```

Comment:
Can we call the event `abnormal_termination_event` or `non_zero_exit_event`? Crash is short but maybe not expressing what it encodes.

Reply:
Sure, pick a name and I'll change it.  Maybe  `worker_failed_event`?

Reply:
Let's call it `terminate_all_event`, so it describes its effect, not its cause.